charts/vlm/values.yaml

```diff
@@ -14,8 +14,9 @@ podAnnotations: {}
 deploymentAnnotations: {}
 
 service:
-  type: ClusterIP
+  type: NodePort
   port: 6000
```

Comment:
slightly off-topic, but the `pipeline-runner` is on `6000`.  I think everything will work with port conflicts but I'm not 100% sure... should we change it during development?

Reply:
oh good catch, I will change this everywhere